src/az-databricks/deploy/bicep/network.bicep

```diff
@@ -1,51 +1,62 @@
-@description('The base name to be appended to all provisioned resources.')
-@maxLength(13)
-param baseName string
+@description('The name of the Azure Databricks virtual network.')
+param workspaceVnetName string
 
-@description('The name of the virtual network.')
-param vnetName string
+@description('The virtual network IP space to use for the Azure Databricks workspace virtual network')
+param workspaceVnetAddressPrefix string
 
-@description('The name of the virtual network subnet to be used for private endpoints.')
-param peSubnetName string
+@description('The name of the virtual network subnet to be used for databricks workspace backend private endpoint.')
+param workspaceBackendPrivateEndpointSubnetName string
 
-@description('The name of the databricks host subnet.')
-param hostSubnetName string
+@description('The IP space to use for the subnet for databricks workspace private endpoint.')
+param workspaceBackendPrivateEndpointSubnetAddressPrefix string
 
-@description('The name of the databricks container subnet.')
-param containerSubnetName string
+@description('The name of the databricks workspace host subnet.')
+param workspaceHostSubnetName string
 
-@description('The name of the network security group to be attached to the databricks subnets.')
-param networkSecurityGroupName string
+@description('The IP space to use for the databricks host subnet.')
+param workspaceHostSubnetAddressPrefix string
 
-@description('The virtual network IP space to use for the new virutal network.')
-param vnetAddressPrefix string
+@description('The name of the databricks workspace container subnet.')
+param workspaceContainerSubnetName string
 
-@description('The IP space to use for the subnet for private endpoints.')
-param privateEndpointSubnetAddressPrefix string
+@description('The IP space to use for the databricks container subnet.')
+param workspaceContainerSubnetAddressPrefix string
 
-@description('The IP space to use for the AzureBastionSubnet subnet.')
-param bastionSubnetAddressPrefix string
+@description('The name of the network security group to be attached to the databricks workspace subnets.')
+param workspaceNetworkSecurityGroupName string
 
-@description('The IP space to use for the databricks host subnet.')
-param hostSubnetAddressPrefix string
+@description('The name of the transit virtual network.')
+param transitVnetName string
 
-@description('The IP space to use for the databricks container subnet.')
-param containerSubnetAddressPrefix string
+@description('The virtual network IP space to use for the transit virtual network')
+param transitVnetAddressPrefix string
+
+@description('The name of the virtual network subnet to be used for private endpoints.')
+param transitPrivateEndpointSubnetName string
+
+@description('The IP space to use for the subnet for private endpoints.')
+param transitPrivateEndpointSubnetAddressPrefix string
+
+@description('The name of the AzureBastionSubnet subnet.')
+param transitBastionSubnetName string
```

Comment:
Could this have a default value of "AzureBastionSubnet"?  Is that still the required subnet name for the Bastion service?